던전 각 계층 기반 구현

dedaia opened the pull request:
던전의 기본인 각 계층의 기반 시스템 구현

기반 시스템을 이용하여 1층, 2층, 3층의 시작방 밑 방들을 블루프린트로 구현

몬스터들을 다 쓰러트리고 다음 방으로 이동할 경우 자동으로 다음 방이 스폰되도록 하는 자동 스폰 시스템 구현

방의 크기 조절시 변수 하나를 수정하여 조절된 방 크기에 맞추어 자동으로 방 스폰이 원활하게 이루어지도록 자동 시스템 구현

dedaia committed:
던전 블록 클래스 추가 및 1층 매니저 추가

각 던전들의 방을 구성하는 블록에 관련된 클래스 및 블루프린트 추가

1층 던전을 관리하는 1층 던전 매니저 추가

dedaia committed:
.

dedaia committed:
기존 클래스 제거, 던전 기본 클래스 생성

dedaia committed:
04 28 작업분

벽과 바닥부분을 코드로 작성후 메테리얼까지 입혀 레벨에 출력되는 부분까지 확인완료

dedaia committed:
던전 시스템 구축

1층부터 3층까지 던전의 기본이 되는 맵의 제작 및 랜덤 생성 시스템 기반 구축

dedaia committed:
thirdPerson 파일 삭제

LeeTaehuny commented:
1. Tick 내부 DoorUpDown 함수에서 문이 언제나 새로운 위치로 Set되고 있습니다. 목적 위치에 도달하면 더이상 호출되지 않았으면 좋겠습니다.
2. 부모 클래스의 함수를 사용할 때 네임스페이스를 앞에 붙여 사용하는 경우와 그냥 사용하는 경우가 있는데 통일해주시면 좋겠습니다.
3. 헤더 파일에서 bool 변수는 메모리의 효율성을 위해 "uint8 변수명 : 1;"로 선언하기로 이야기 했습니다. 해당 부분 수정 부탁드립니다.
4. 코드에 주석이 없어 내용을 읽기 난해합니다. 간단한 설명이라도 적어주시면 좋겠습니다.

dedaia committed:
피드백 반영

1. 부모 클래스의 함수를 사용할 때 네임스페이스를 사용하지 않는 방식으로 코드 통일

2. 헤더 파일에 선언된 bool 변수를 uint8 변수명 : 1로 모두 변경

3. 헤더에 간단한 코드 설명 주석 추가

LeeTaehuny committed:
Merge branch 'develop' into feature/MonsterAnim
